Makefile

```diff
@@ -10,6 +10,7 @@ endif
 
 .PHONY: clean
 clean:
+	docker-compose down
```

Comment:
Nice, that also removes the network. Should we add `-v` to also remove the volumes?